client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -128,6 +125,35 @@ class SiteIndicator extends Component {
 			);
 		}
 
+		if ( siteUpdates.themes === siteUpdates.total && site.canUpdateFiles ) {
+			return (
+				<span>
+					<a onClick={ this.handleThemesUpdate } href={ activityLogPath }>
+						{ translate(
+							'There is a theme update available.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'There is an update available.', 'There are updates available.' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/26123-update-site-indicator)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).